### DISCUSSION TITLE
Base points position requirements?

### DISCUSSION THREAD

**Discussion by dan-adi** [mworion/MountWizzard4#341] [Q&A] [answered]
Hello,
Are there specific requirements for the first 3 base points in the model? Can they be in the same hemisphere ? 2 on one hemisphere, 1 on the other? To the need to be on the same DEC ?
The manual is not clear to me on this matter.

**Comment by Ruediger3** [comment, marked as answer]
Hello Dan,

When building a model, all points a are treated identically. There are no base points. 

When doing polar alignment, the three points should be spread as far as possible, with the rules stay away from horizon, median and zenith by some degree. But this is not the best way. 

Best practice is to do polar alignment with a small model e.g. 10 points with equal split to both hemispheres. There you get the PA, do the corrections on the knobs, clear the model and repeat it until you have achieved the desired PA. This is by far more precise than any 3 point alignment. This procedure is also recommended by 10M. 

CS
Rüdiger

**Reply by mworion** [reply]
Hi Dan, just as addition: 
MW4 does not treat the points differently at any time. You need at least 3 Points to build a first model technically.
Michel

**Reply by dan-adi** [reply]
I'm not asking specifically for Polar alignment. I'm just interested if the position of the first 3 points matters at all. If they are just like any other points then it should not matter.
I haven't got to use MW4 yet, just the integrated modeling feature in Prism v.11, since I use it for the last 5 years with my older mount and I'm accustomed to it.
Yesterday I was looking carefully at the Prism model terms, and I noticed it displays 11 terms just like 10M uses and the 3 base points had the highest error. Seems natural since they are the first points, and they should have higher errors. One mistake I probably did is place the first 3 points very far apart.
I will post a thread on 10M forum since with Prism the user can evaluate the 11 terms in the model to better troubleshoot problems.
Thanks, have a nice day
